src/chunk_list.cpp

```diff
@@ -51,6 +51,21 @@ chunk_t *chunk_get_tail(void)
 }
 
 
+/**
+ * \brief Searches for a non-vbrace chunk
+ *
+ * follows a chunk list until it finds a non-vbrace chunk.
+ * The search direction is determined by parameter dir.
+ *
+ * @return pointer to the found chunk or NULL of none found
+ */
+static chunk_t *chunk_get_nvb(chunk_t           *cur,           /**< [in] chunk to start with*/
```

Comment:
Wouldn't it be better to describe the params with `@param` inside the description comment?